src/pages/register/register.ts

```diff
@@ -30,9 +41,25 @@ export class RegisterPage {
     console.log('ionViewDidLoad RegisterPage');
   }
 
-  regForm(){
+  submitReg() {
```

Comment:
This could use some formatting--the indenting and format look at bit off and make it harder to read.